p/themes/Origine/origine.css

```
@@ -330,6 +330,8 @@ a.btn {
 	box-shadow: 3px 3px 3px #ddd;
 	font-size: 0.8rem;
 	text-align: left;
+	max-height: 60vh;
```

```suggestion
	max-height: 75vh;
```
60 seems really tiny to me?

---

yea … ok … i still can apply a personal custom stylesheet if i personally want it smaller